crates/bevy_app/src/plugin_group.rs

```diff
@@ -23,6 +23,22 @@ pub struct PluginGroupBuilder {
 }
 
 impl PluginGroupBuilder {
+    /// Finds the index of a target [`Plugin`]. Panics if the target's [`TypeId`] is not found.
+    fn index_of<Target: Plugin>(&mut self) -> usize {
+        let index = self
+            .order
+            .iter()
+            .position(|&ty| ty == TypeId::of::<Target>());
+
+        match index {
+            Some(i) => i,
+            None => panic!(
+                "Plugin does not exist: {}.",
```

Comment:
```suggestion
                "Plugin does not exist in group: {}.",
```
Or better phrasing. The plugin may exists, just not in this group.

Reply:
agreed. this has been updated